build-utils/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@reshuffle/build-utils",
+  "version": "0.0.1",
+  "description": "Build utils for reshuffle",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "rm -rf dist/ && tsc",
+    "lint": "eslint -c ../common/.eslintrc.json --ext .ts src",
+    "test": "echo No tests yet"
+  },
+  "files": [
+    "/dist"
+  ],
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@reshuffle/utils-subprocess": "~0.0.2",
```

Comment:
Nit^extreme: can we haz carets?

Reply:
changed
`rush update` didn't create changes, is that alright?

Reply:
I think it's OK.

Also for `@reshuffle/utils-subprocess` specifically should be exact `"0.0.2"` since rush will manage the exact dependency when upgrading.